package.json

```diff
@@ -87,5 +93,6 @@
   ],
   "dependencies": {
     "lodash": "4.17.21"
-  }
+  },
+  "type": "module"
```

Comment:
this shouldn't be necessary here. This is a mono-repo and everything is bundled by vite. 

Look inside the `packages/` directory.